interface/src/app/gene/network/network.js

```diff
@@ -187,20 +205,22 @@ angular.module('adage.gene.network', [
           NodeService.get(
             {'heavy_genes': heavyGenes, 'limit': 0},
             function success(response) {
-              var n = response.objects.length;
+              var i = 0, n = response.objects.length;
+              var uri;
```

Comment:
Now that this code is building a `uri` alongside the `str`, should we give `str` a more descriptive name? Maybe something as simple as `tipStr` would make this bit of code a little easier to follow.